SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day3/hashset/ExerciseHashset.java

```diff
@@ -0,0 +1,34 @@
+package hackerrank.day3.hashset;
+
+import java.util.HashSet;
+import java.util.Scanner;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 31/08/21
+ */
+public class ExerciseHashset {
+    /**
+     * Get names and verify duplicates
+     */
+    public static void getScanNames() {
+
+        Scanner scanner = new Scanner(System.in);
+        int testCases = scanner.nextInt();
+        String[] pairLeft = new String[testCases];
+        String[] pairRight = new String[testCases];
+
+        for (int index = 0; index < testCases; index++) {
+            pairLeft[index] = scanner.next();
+            pairRight[index] = scanner.next();
+        }
+
+        HashSet<String> pairs = new HashSet<>();
```

Comment:
remember Code for Interface, not for Implementation:
https://www.javaguides.net/2018/06/java-collection-framework-best-practices.html